### PR TITLE
Feature/iesp 65

### PR DESCRIPTION
**Pull request by AlessandroColucci**
This pr regards the changes required for the application to work on the developer environment. Thats why both me and Alessio Creo worked on it.

### CODE REVIEW

**Review by nttdata-rtorsoli — APPROVED** [review]

File: src/main/java/it/pagopa/interop/probing/eservice/registry/reader/config/PropertiesLoader.java [hunk 1]
```diff
@@ -36,29 +35,15 @@ public class PropertiesLoader {
...
-			this.props.load(inputStream);
-		} catch (IOException e) {
-			log.error("Error during reading properties from file");
-			throw e;
-		} catch (NullPointerException e) {
-			log.error("Error during reading properties from file");
-			throw e;
-		}
+	public PropertiesLoader() {
+		this.props = System.getenv();
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/eservice/registry/reader/config/PropertiesLoader.java:46]: isn't it possible to keep the `application.properties` file?
taking values blindly directly from env makes really difficult to understand which configurations are required for the application

**Review by galales — CHANGES_REQUESTED** [review]

File: src/main/java/it/pagopa/interop/probing/eservice/registry/reader/config/PropertiesLoader.java (hunk 1 above)
Comment (AlessandroColucci) [reply src/main/java/it/pagopa/interop/probing/eservice/registry/reader/config/PropertiesLoader.java:46]: In a java main application we do not have a container that injects the variables into the secrets, so we decided to set the variables via kubernetes into the env. Maybe we can group all the keys used to take the variables into a single file, in order to be more clear.

Comment (galales) [reply src/main/java/it/pagopa/interop/probing/eservice/registry/reader/config/PropertiesLoader.java:46]: setting kubernetes secrets into application env variables is ok, and it should be similar to the previous approach, where the `PropertiesLoader` loaded env vars based on `application.properties`.
Or am I missing something?

**Review by micdes-pagopa — APPROVED** [review]
Approve on devops part of the PR

**Review by galales — APPROVED** [review]

**Pull request merged**